tests/config/test_model_config.py

```
@@ -0,0 +1,77 @@
+    return ModelConfig(
+                        init_weight="default", 
+                        pre_trained_weights=None,
+                        backbone_config=BackboneConfig(),
+                        head_configs=HeadConfig()
+                        )
+
+
+def test_default_initialization(default_config):
+    """Test default initialization of ModelConfig."""
+    assert default_config.init_weight == "default"
+    assert default_config.pre_trained_weights == None
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)